### ISSUE TITLE
Windows App Install Creates Shortcut Under 'GitHub, Inc' Programs Group

### ISSUE THREAD

**Issue by schellem** [a85/POSTMan-Chrome-Extension#2816]
I installed the x64 Windows app on Windows 8.1 machine using 'Classic Start Menu' and it created no shortcut nor could I find any trace of in either Program Files. After much investigation I find a shortcut under "Github, Inc".....and a very odd one at that C:\Users\user\AppData\Local\Postman\Update.exe --processStart "Postman.exe"

You should make the app easier to find after the install.

**Comment by sdnts** [comment]
@schellem Ah, thanks for pointing this out. We use GitHub's Electron for our native apps, which must be causing this.

**Comment by kamalaknn** [comment]
@schellem  Thanks for pointing this out to us. This was a result of an incorrect configuration from our side.

We have updated this in version 4.10.5. New installs will now create shortcut under "Postman" directory in start menu.

Closing this now. Feel free to re open if it still exists.

**Comment by harryi3t** [comment]
Reopening this since multiple people reported that this is still happening

user reports:
 - Twitter: https://twitter.com/web2033/status/1169500436086317056
 - GitHub: https://github.com/postmanlabs/postman-app-support/issues/7186

**Comment by harryi3t** [comment]
Affected versions: >= v7.3.4

**Comment by harryi3t** [comment]
This is fixed in Postman v7.7.2
Let us know if the issue persists.

**Comment by uvarvu** [comment]
@harryi3t It's not fixed in the canary stream.
EDIT:   It's not fixed in 7.7.2 either.

**Comment by harryi3t** [comment]
@uvarvu We have made the fix to not create new shortcuts in the `GitHub, Inc` directory on fresh installs and on updates. If that is still happening please let us know.

However, this does not automatically remove shortcuts created by previous versions under `GitHub, Inc` directory. We are working on a fix that will clean up these shortcuts. We want to be sure about the side effects of the cleanup before shipping it.

To unblock yourself now:
 1. make sure you are on the latest version of Postman (v7.7.2 or 7.8.0-canary02)
 2. verify that a new shortcut is present in `Start menu -> Postman` directory
 3. delete the old shortcut inside the `Start menu -> GitHub, Inc` directory manually.

**Comment by uvarvu** [comment]
@harryi3t Ahh apologies, I was linked here from another issue regarding the app icon not appearing in apps and features on Windows.   I can confirm the directory location is indeed fixed but the app icon issue is still there.

**Comment by harryi3t** [comment]
@uvarvu I have replied here https://github.com/postmanlabs/postman-app-support/issues/6897#issuecomment-532167201
